### ISSUE TITLE
Where is subset.py?

### ISSUE THREAD

**Issue by QL111111** [junqiangchen/KiTS19-Challege#12]
Thanks for your contribution.
I noticed that the fifth point in ‘Preprocess’ is split trainSegmentation.csv into training set and test set, but I can't find subset.py in your code.
What should I do, or can you update 'How to Use'?

**Comment by junqiangchen** [comment]
subset.py in the project of dataprocess folder,pls check out it,and how to use it,you can find the readme.
